public/locales/en-US.json

```diff
@@ -205,13 +220,25 @@
     "header_Instance-hours": "Instance hours",
     "header_measurementType": "Type",
     "header_measurement_type": "Type",
+    "header_number_of_guests": "Guests",
     "header_inventoryId": "UUID",
+    "header_Sockets": "Sockets",
+    "header_Sockets_RHEL": "Capacity (sockets)",
+    "header_Sockets_RHEL for ARM": "$t(curiosity-inventory.header_Sockets_RHEL)",
+    "header_Sockets_RHEL for IBM Power": "$t(curiosity-inventory.header_Sockets_RHEL)",
+    "header_Sockets_RHEL for IBM z": "$t(curiosity-inventory.header_Sockets_RHEL)",
+    "header_Sockets_RHEL for x86": "$t(curiosity-inventory.header_Sockets_RHEL)",
     "header_sockets": "Subscribed sockets",
     "header_sockets_OpenShift Container Platform": "Sockets",
     "header_sockets_OpenShift-metrics": "Sockets",
     "header_sockets_OpenShift-dedicated-metrics": "Sockets",
-    "header_lastSeen": "Last seen",
+    "header_lastSeen": "Usage updated",
     "header_last_seen": "Last seen",
+    "header_last_seen_RHEL": "Usage updated",
+    "header_last_seen_RHEL for ARM": "$t(curiosity-inventory.header_last_seen_RHEL)",
+    "header_last_seen_RHEL for IBM Power": "$t(curiosity-inventory.header_last_seen_RHEL)",
+    "header_last_seen_RHEL for IBM z": "$t(curiosity-inventory.header_last_seen_RHEL)",
+    "header_last_seen_RHEL for x86": "$t(curiosity-inventory.header_last_seen_RHEL)",
```

Comment:
required: remove rhel archs, no longer needed with the filter PR #1014  that merged